examples/plot_simulate_somato.py

```diff
@@ -4,8 +4,8 @@
 ================================================
 
 This example demonstrates how to calculate the inverse solution of the median
-nerve evoked response in the MNE somatosensory dataset, and then simulate a
-biophysical model network that reproduces the observed dynamics.
+nerve evoked response in S1 from the MNE somatosensory dataset, and then
```

Comment:
Added "S1 from" to specify that our method now extracts the evoked response specifically from S1.